Workbooks/WorkbooksMetadata.json

```diff
@@ -797,5 +797,20 @@
     "templateRelativePath": "SysmonThreatHunting.json",
     "subtitle": "",
     "provider": "Azure Sentinel community"
+  },
+  {
+  "workbookKey": "CyberArkWorkbook",
+  "logoFileName": "CyberArk_Logo.svg",
```

Comment:
Please check in this logo file in the images -> logo folder per the workbook guidance https://aka.ms/sentinelgithubworkbooks